tests/data/github/github_events

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Do you think it would be better to create a new directory `githubql` to store this data?

Reply:
I understand that the test data is part of GitHub, so it can make sense to leave in the same folder. However if you find it convenient, we can move it to a separate folder.